cc_dv_bm3024.Rmd

```diff
@@ -0,0 +1,190 @@
+# Plotly R graphing
+
+Brian Mao
+## Introduction
+
+Plotly's R graphing library makes interactive, publication-quality graphs. in this into to Plotly I will show some examples of how to make line plots, scatter plots, area charts, bar charts, box plots, histograms, heatmaps, and 3D scatter plot charts. there are a lots more advanced function in the Plotly. you may explore it later on yourself 
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## URL
+
+URL https://plotly-r.com, https://github.com/ropensci/plotly#readme,
+https://plotly.com/r/
+
+## Install and packages
+
+```{r message=FALSE}
+library(plotly)
+library(ggplot2)
+```
+
+## Basic examples
+
+### Basic Scatter Plot
+
+```{r message=FALSE}
+library(plotly)
+fig <- plot_ly(data = iris, x = ~Sepal.Length, y = ~Petal.Length)
+fig
+```
+
+### Plotting Markers and Lines
+
+```{r message=FALSE}
+library(plotly)
+trace_0 <- rnorm(100, mean = 5)
+trace_1 <- rnorm(100, mean = 0)
+trace_2 <- rnorm(100, mean = -5)
+x <- c(1:100)
+data <- data.frame(x, trace_0, trace_1, trace_2)
+fig <- plot_ly(data, x = ~x)
+fig <- fig %>% add_trace(y = ~trace_0, name = 'trace 0',mode = 'lines')
+fig <- fig %>% add_trace(y = ~trace_1, name = 'trace 1', mode = 'lines+markers')
+fig <- fig %>% add_trace(y = ~trace_2, name = 'trace 2', mode = 'markers')
+fig
+```
+
+
+### Data with different Symbols
+
+```{r message=FALSE}
+library(plotly)
+fig <- plot_ly(data = iris, x = ~Sepal.Length, y = ~Petal.Length, type = 'scatter',
+  mode = 'markers', symbol = ~Species, symbols = c('circle','x','o'),
+  color = I('black'), marker = list(size = 10))
+fig
+```
+
+## Statistical Charts
+
+#### Basic Boxplot
```

Comment:
On line 61 the heading has 2 hashtags, line 63 must have 3 hashtags for correct ordering.